core/src/executor/execute.rs

```diff
@@ -36,6 +36,9 @@ pub enum ExecuteError {
 
 #[derive(Serialize, Debug, PartialEq)]
 pub enum Payload {
+    ShowColumns {
+        rows: Vec<(String, DataType)>,
```

Comment:
The command is `SHOW COLUMNS`, then how about using `columns` rather than `rows`?
Or, this might be enough to declare with `ShowColumns(Vec<(String, DataType)>)` because it has an single value and this looks obvious that the value inside `ShowColumns` has `columns`.

Reply:
+1